src/pages/journal/components/empty-slot/empty-slot.scss

```diff
@@ -0,0 +1,36 @@
+empty-slot {
+  ion-card {
+    &.grey-bg {
```

Comment:
realise I suggested this name, but its not really a grey bg! can we re-name to transparent-bg?

Reply:
I agree - transparent-bg is a much better name will change now.